hazelcast/src/main/java/com/hazelcast/internal/ascii/rest/HttpPostCommandProcessor.java

```diff
@@ -323,6 +325,25 @@ private void handleWanSyncMap(HttpPostCommand command) throws UnsupportedEncodin
         textCommandService.sendResponse(command);
     }
 
+    private void handleWanClearQueues(HttpPostCommand command) throws UnsupportedEncodingException {
```

Comment:
Yes but WAN replication is paused/resumed automatically when clear queues command is in progress.